code/modules/jobs/job_types/_job.dm

```diff
@@ -271,7 +271,7 @@
 			qdel(skill_chip)
 			return
 
-		var/activate_msg = skillchip.try_activate_skillchip(TRUE, TRUE)
+		var/activate_msg = skill_chip.try_activate_skillchip(TRUE, TRUE)
```

Comment:
If you could rename `skillchip` to `skillchip_type`, that'd be ideal.

Reply:
I committed a full renaming sweep before I saw this review. Are the new names acceptable?